### ISSUE TITLE
Femtovg Border radius bug 

### ISSUE THREAD

**Issue by NigelBreslaw** [slint-ui/slint#6506]
Noticed a difference in rendering of border-radius on WASM. In the image the incorrect and squished looking slider is caused by its border-radius being set to `self.height`. Once set to `self.height / 2` it looks fine. However it should auto clamp to that value and not render differently on WASM?
<img width="260" alt="Screenshot 2024-10-10 at 11 08 07" src="https://github.com/user-attachments/assets/a44de2bb-669b-45d0-8d72-b1bab2b5d4a4">
<img width="240" alt="Screenshot 2024-10-10 at 11 07 38" src="https://github.com/user-attachments/assets/13b0e993-d611-4b74-824a-57f9b461d0cd">

**Comment by fivlao** [comment]
I also came across this bug in my project
Code:
```Rust
/*
# Cargo.toml
[package]
name = "tests"
version = "0.1.0"
edition = "2024"

[dependencies]
slint = "1.15.1"
*/


slint::slint! {
    export component App inherits Window {
        width: 800px /2;
        height: 600px /2;

        Rectangle {
            width: 300px;
            height: 75px;
            background: black;
            border-radius: self.height;
        }

    }
}

fn main() {
    let app = App::new().unwrap();
    app.run().unwrap()
}

```

<img width="703" height="858" alt="Image" src="https://github.com/user-attachments/assets/56b45c9b-f1c2-4545-bc83-fd41de97db1b" />

**Comment by tilladam** [comment]
Checked where this stands: the FemtoVG-side fix exists but has not been released yet.

[femtovg#307](https://github.com/femtovg/femtovg/pull/307) ("Scale rounded rect radii together instead of per axis", merged 2026-07-27) is the relevant change, but femtovg **v0.26.0 was tagged 2026-07-20** — a week earlier — and no release has been cut since. Comparing the merge commit against every existing tag puts it *ahead* of all of them, so it is in no published version.

Slint currently pins `femtovg = 0.26.0`, so this cannot be fixed by a dependency bump today. It is blocked on a femtovg release containing #307; once one exists, bumping should be enough.

**Comment by tronical** [comment]
Yep, I had making release on my todo after I fixed it in femtovg. I'll pin this to our release just to be sure to close this.
